src/Config.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Netzstrategen\MultiStock;
+
+/**
+ * Plugin settings functionality.
+ */
+class Config {
+
+  /**
+   * Config JSON file.
+   *
+   * @var string
+   */
+  const CONFIG_FILE = '/config.json';
```

Comment:
@sun + @marcbinz — I'm not sure how handle the `config.json` path here.

I was thinking that in some cases we will need to handle with different files in the same repo (e.g. WOPA/LART). Perhaps is a better idea to define the config file in the `wp-config.php` and use `config.json` as fallback? 🤔 
What do you think?

Also, perhaps instead of placing the `config.json` file in the plugin root, we should move it to the related website `wp-content/uploads`?

Reply:
As soon as we decide an approach I will update the `README.md` file to clarly state how to setup/test the plugin.

Reply:
@LucaPipolo I am not completely sure what the Wordpress convention is here, but I think the `uploads` folder sounds wrong for placing config files there. So I would rather set an ENV or constant via `wp-config.php` to a global config folder or something 🙂

Reply:
We should move config file into document root (i.e., ABSPATH) (optionally supporting a constant in wp-config.php specifying a custom location, but this is not necessary yet).

We should also rename the config file name to something less generic; e.g., `ABSPATH . 'stores.json`

Consider to prefix the filename with a dot, so that it is treated like a hidden dotfile (which should not be served to the public).